selfdrive/car/gm/interface.py

```diff
@@ -198,6 +198,16 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long):
       ret.steerRatio = 15.3
       ret.centerToFront = ret.wheelbase * 0.5
 
+    elif candidate == CAR.ESCALADE:
+      ret.minEnableSpeed = -1.  # engage speed is decided by pcmFalse
+      ret.mass = 5653. * CV.LB_TO_KG + STD_CARGO_KG  # (5552+5815)/2
+      ret.wheelbase = 2.95  # 116 inches in meters
+      ret.steerRatio = 17.3
+      ret.centerToFront = ret.wheelbase * 0.5
+      ret.longitudinalTuning.kpV = [3.5, 2.18]
```

Comment:
does the global tune not work very well?

Reply:
It didn't at first, but that may no longer be the case. I'll do a test run without it and report back.

Reply:
what was the problem with it? late braking/acceleration? let me know if you think it's good enough and we can merge!

Reply:
Late braking was the main issue. Just did a test run without it though and acceleration/stopping both seemed reasonable. 

Should be good to merge as far as I'm concerned!

Reply:
You can do something like this for the Acadia: https://github.com/commaai/openpilot/blob/89934b35d04291384b7a287a41ce49905f321167/selfdrive/car/gm/interface.py#L187

Seems like all GMs without regen control have a high delay to start braking. ICE and EVs without regen like the newer Bolt EVs

Reply:
Your longitudinal looks very similar to @Verylukyguy's. Try out that tuning option! In the mean time, this PR looks good to go. Thanks!

Reply:
Ah, thank you! I'll give that a try in a few days when I have the bandwidth to do a proper check with PlotJuggler vs a more subjective check. 

If there's a significant change I can always open another PR in the future. 

It seems solid as is though either way. I didn't have to take over braking over about a 25 minute drive, so that's a significant improvement over when I initially tried to tune.

Reply:
Curious, how is steering? The plots look rather good. No oscillation, but could be a bit lazy it looks like.

Reply:
Overall good. We took a 9 hour drive today and needed to help it on curves while at highway speed, and there was some minor oscillation instances where the road was sloped at an angle, but overall centered well!